lib/has_crud/has_crud/active_record.rb

```diff
@@ -179,6 +180,12 @@ def setup_composable
         end
       end
 
+      def setup_has_managed_assets
+        if self.options[:has_managed_assets]
+          send :include, HasManagedAssets
+        end
+      end
```

Comment:
@dbaines Updated so that we now specify `has_crud has_managed_assets: true` as per other koi modules, rather than including the concern directly. (Obviously you can still include the concern directly if you want!)